docs/install/command-line.md

```diff
@@ -89,43 +89,69 @@ The IPFS team manages the [dist.ipfs.tech website](https://dist.ipfs.tech/) to h
 
 ### macOS
 
-:::tip M1-based Macs
-You can install IPFS on M1-based Macs by using the `darwin-arm64` binary instead of the `amd64` binary listed in these instructions.
-:::
+1. Decide which macOS binary to download from [`dist.ipfs.tech`](https://dist.ipfs.tech/#kubo) by determining whether your system uses Apple or Intel CPUs. On most MacOS systems, you can determine the system specs by doing the following:
 
-1. Download the macOS binary from [`dist.ipfs.tech`](https://dist.ipfs.tech/#kubo).
+   a. In the upper left hand corner of your screen, click the "Apple" icon. 
+   b. In the pop-up menu displayed, select `About this Mac`.
+   c. Information on your Mac system is displayed. 
+      - If your system uses Apple Silicon, the specific `Chip` is shown, such as `Apple M1 Pro`.
+      - If your system uses an Intel CPU, the specific `Processor` is shown, such as `2.3 GHz 8-Core Intel Core i9`.
```

Comment:
Remove the backticks from around "Chip" and "Processor". I don't think they need quotation marks either.

Reply:
Addressed in https://github.com/ipfs/ipfs-docs/pull/1289/commits/6c3449285abbdad3be0adf67a23685bcc93457aa

Reply:
Again, based on my comment above I now think "Chip" and "Processor" should just be **bold**.